NuGet.Config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+    <add key="azure-sdk-for-net" value="https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-net/index.json" />
```

Comment:
I suggest using the Azure Devops feed like .NET, see https://github.com/Azure/azure-sdk-for-net/blob/master/NuGet.Config#L8.

Reply:
Sorry I just realized that the .NET nuget.config doesn't have the devops feed yet either. The feed I suggest using is 
```
<add key="azure-sdk-for-net" value="https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json" />
```

Reply:
.NET repo update is still in PR https://github.com/Azure/azure-sdk-for-net/pull/11916.

Reply:
Fixed here - https://github.com/Azure/autorest.csharp/pull/826